ros/src/foxglove_msgs/.gitignore

```diff
@@ -0,0 +1 @@
+/msg/
```

Comment:
Just curious as to why these .gitignores are being introduced as separate files, and not lines in the top-level .gitignore?

Reply:
no real reason, this one was already here I just gave it a `/` prefix. this particular one is going to go away once we drop support for ros1 which I intend to propose next.